src/components/Utils/resources.js

```diff
@@ -184,3 +186,25 @@ export const ORDER_NUMBER_SETTING = {
   },
   ...BASE_RESOURCE,
 };
+
+export const PREFIXES_SETTING = {
+  path: CONFIG_API,
+  records: 'configs',
+  GET: {
+    params: {
+      query: `(module=${MODULE_ORDERS} and configName=${CONFIG_PREFIXES})`,
+    },
+  },
+  ...BASE_RESOURCE,
```

Comment:
please put this always at the top to be able redefine the values

Reply:
Fixed